kythe/cxx/indexer/cxx/IndexerFrontendAction.cc

```diff
@@ -216,6 +216,7 @@ std::string IndexCompilationUnit(
   options.build_config = ExtractBuildConfig(Unit);
   options.default_corpus =
       Options.UseCompilationCorpusAsDefault ? Unit.v_name().corpus() : "";
+  options.hash_recorder = std::move(Options.HashRecorder);
```

Comment:
The caller didn't transfer ownership of the options struct, stealing from its guts is inappropriate.

This should either be passed by value or the HashRecorder ownership be retained by the caller.